assets/messages.yml

```
@@ -1039,6 +1039,19 @@ messages:
       shortcut: qul
       message: "%quick_links%"
       fillname: []
+  removed-version:
+    - project:
+        - mc
+        - mce
+        - mcl
+        - mcpe
+      name: Panel - Removed Extra Version
+      shortcut: remver
+      message:
+        "{panel:borderColor=orange}(!) Please do not add Affected Versions
+        to resolved reports.
```

Might be good to add that the user should look at the Resolution and the comments, otherwise they might not understand what "resolved" means and might think the issue they reported has been marked as fixed.
```suggestion
        to resolved reports. Have a look at the Resolution and the comments to see why this ticket has been resolved.
```